evi-typescript-example/package.json

```diff
@@ -9,7 +9,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@humeai/voice": "0.0.0-beta.20"
+    "@humeai/voice": "^0.1.2"
```

Comment:
@jerhadf We can remove the `evi-typescript-example` project changes from this branch. Some of these changes are already in. I am currently working on it in another branch to utilize the Fern typescript SDK.

Reply:
okay sg could you remove those changes please? 🙏 sorry working on some other pre-release tool use things

Reply:
For sure!